src/main/java/net/dv8tion/jda/internal/entities/TextChannelImpl.java

```diff
@@ -458,14 +458,23 @@ public MessageAction sendFile(@Nonnull InputStream data, @Nonnull String fileNam
         Checks.isSnowflake(messageId, "Message ID");
         Checks.noWhitespace(unicode, "Unicode emoji");
         Checks.notNull(user, "User");
+
```

Comment:
The `noWhitespace` check should be changed, see other comment above

Reply:
Gotcha.